go.mod

```diff
@@ -1,9 +1,6 @@
 module github.com/stackrox/rox
 
 go 1.21
-
-toolchain go1.21.7
```

Comment:
This should not happen https://github.com/dependabot/dependabot-core/issues/9422

Reply:
The package declares newer go version than we use: https://github.com/Shopify/toxiproxy/blob/v2.9.0/go.mod#L3
What do we do with it?

Reply:
https://redhat-internal.slack.com/archives/C02CFR7GQUS/p1714740069768599